### PR TITLE
Update mdo-email-entity-page.md

### PR DESCRIPTION
**Pull request by MishraSoumyaMS**
https://github.com/MishraSoumyaMS/microsoft-365-docs/commit/87e8ec10ef2bdb353034b5b4b9d96b1a897e2a48 reference images

### CODE REVIEW

**Review by MSFTTracyP — APPROVED** [review]

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 1]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
   - None: Indicates that no DMARC TXT record exists for the sending domain in DNS.
 
 *Composite Authentication*: This is a value used by Microsoft 365 to combine email authentication like SPF, DKIM, and DMARC, to determine if the message is authentic. It uses the *From:* domain of the mail as the basis of evaluation.
+## Actions into Email entity Page
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:164]:
```suggestion
## Actions into mail entity page
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 2]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
...
 
+Security teams can now take email actions like soft delete and hard delete, move to junk, move to inbox, trigger an investigation, submit to Microsoft for review in line, and et cetera. **Tenant level block** actions like file and URL or sender can also be trigged from Email entity page.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:166]:
```suggestion
Security teams can now take email actions like soft delete and hard delete, move to junk, move to inbox, trigger an investigation, submit to Microsoft for review in line, and more. **Tenant level block** actions like file and URL or sender can also be triggered from the email entity page.  
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 3]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
   - None: Indicates that no DMARC TXT record exists for the sending domain in DNS.
 
 *Composite Authentication*: This is a value used by Microsoft 365 to combine email authentication like SPF, DKIM, and DMARC, to determine if the message is authentic. It uses the *From:* domain of the mail as the basis of evaluation.
+## Actions into Email entity Page
 
+Security teams can now take email actions like soft delete and hard delete, move to junk, move to inbox, trigger an investigation, submit to Microsoft for review in line, and et cetera. **Tenant level block** actions like file and URL or sender can also be trigged from Email entity page.  
+
+You will be able to click on **Take actions** from the top right corner of the entity page and this will open the Action wizard for you to select the specific action you need.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:168]:
```suggestion
Select **Take actions** from the top right corner of the entity page to open the Action wizard for you to select the specific action you need. 
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 4]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
...
+
+You will be able to click on **Take actions** from the top right corner of the entity page and this will open the Action wizard for you to select the specific action you need. 
+![Take action from entity page.](../../media/Take-ActionWizard-Email-entity.png)
+
+In the Action wizard you can take email actions, email submissions, block sender and sender domain, investigative actions and two step approval (add to remediation) in the same side pane. This follows a consistent flow for ease of use. The Action wizard uses the same system as is used by Explorer actions (for Delete, Submissions, and Investigation actions), for example. You will be able to see and track these actions in the 
+ [Unified action center](https://security.microsoft.com/action-center/history) (for deleted emails), in the 
+  [Submission portal](https://security.microsoft.com/reportsubmission) (for submissions), and in [Tenant Allow/Block Lists](https://security.microsoft.com/tenantAllowBlockList) page for (TABL blocks).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:173]:
```suggestion
  [Submission portal](https://security.microsoft.com/reportsubmission) (for submissions), and in the [Tenant Allow/Block Lists](https://security.microsoft.com/tenantAllowBlockList) page (for TABL blocks). 
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 5]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
...
+![Take action from entity page.](../../media/Take-ActionWizard-Email-entity.png)
+
+In the Action wizard you can take email actions, email submissions, block sender and sender domain, investigative actions and two step approval (add to remediation) in the same side pane. This follows a consistent flow for ease of use. The Action wizard uses the same system as is used by Explorer actions (for Delete, Submissions, and Investigation actions), for example. You will be able to see and track these actions in the 
+ [Unified action center](https://security.microsoft.com/action-center/history) (for deleted emails), in the 
+  [Submission portal](https://security.microsoft.com/reportsubmission) (for submissions), and in [Tenant Allow/Block Lists](https://security.microsoft.com/tenantAllowBlockList) page for (TABL blocks). 
+
+We are also bringing Tenant level block URL and attachment to the respective Email entity URL and Attachments tabs. Upon approval, all the Tenant Allow and Block Lists (or TABL) block URL and block attachments can be tracked under TABL/URL and TABL/file pages.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:175]:
```suggestion
Tenant level Block URL and Attachment is also available in the respective Email entity URL and Attachment tabs. Upon approval, all the Tenant Allow and Block lists (TABL) block URLs and block attachments can be tracked under the TABL/URL and TABL/file pages. 
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 6]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
...
+## Actions into Email entity Page
 
+Security teams can now take email actions like soft delete and hard delete, move to junk, move to inbox, trigger an investigation, submit to Microsoft for review in line, and et cetera. **Tenant level block** actions like file and URL or sender can also be trigged from Email entity page.  
+
+You will be able to click on **Take actions** from the top right corner of the entity page and this will open the Action wizard for you to select the specific action you need. 
+![Take action from entity page.](../../media/Take-ActionWizard-Email-entity.png)
+
+In the Action wizard you can take email actions, email submissions, block sender and sender domain, investigative actions and two step approval (add to remediation) in the same side pane. This follows a consistent flow for ease of use. The Action wizard uses the same system as is used by Explorer actions (for Delete, Submissions, and Investigation actions), for example. You will be able to see and track these actions in the
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:171]:
```suggestion
In the Action wizard, you can take email actions, email submissions, block senders and sender domains, investigate actions, and two-step approval (add to remediation) in the same side pane. This follows a consistent flow for ease of use. The Action wizard uses the same system as is used by Explorer actions (for Delete, Submissions, and Investigation actions), for example. You can see and track these actions in the 
```

File: microsoft-365/security/office-365-security/mdo-email-entity-page.md [hunk 7]
```diff
@@ -161,7 +161,23 @@ Users will see enriched detonation details for known malicious attachments or UR
...
+In the Action wizard you can take email actions, email submissions, block sender and sender domain, investigative actions and two step approval (add to remediation) in the same side pane. This follows a consistent flow for ease of use. The Action wizard uses the same system as is used by Explorer actions (for Delete, Submissions, and Investigation actions), for example. You will be able to see and track these actions in the 
+ [Unified action center](https://security.microsoft.com/action-center/history) (for deleted emails), in the 
+  [Submission portal](https://security.microsoft.com/reportsubmission) (for submissions), and in [Tenant Allow/Block Lists](https://security.microsoft.com/tenantAllowBlockList) page for (TABL blocks). 
+
+We are also bringing Tenant level block URL and attachment to the respective Email entity URL and Attachments tabs. Upon approval, all the Tenant Allow and Block Lists (or TABL) block URL and block attachments can be tracked under TABL/URL and TABL/file pages. 
+![Take block URL action from entity page.](../../media/Block-URL-Email-entity.png)
+
+See [permissions](permissions-microsoft-365-security-center.md) required to take these actions. 
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/mdo-email-entity-page.md:180]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MishraSoumyaMS, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by MSFTTracyP — APPROVED** [review]

**Comment by MSFTTracyP** [comment]
Overdue for publish.

**Review by MSFTTracyP — COMMENTED** [review]
Publish Immediately.

**Pull request merged**